src/components/Header.jsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 
 const Header = () => (
```

Comment:
Modify the Header to be a react component with a render method. Currently, the nav menu is missing a method to initialize it for mobile (see the Initialization section at the bottom of http://materializecss.com/navbar.html). While they want you to use jQuery, just use a ref instead and do the initialization in componentDidMount. If you try the Materialize site in a mobile view you'll see that they have the hamburger button which slides out a menu. We should end up with the same thing.